src/main/java/com/walabot/home/ble/sdk/Extensions.kt

```diff
@@ -23,4 +27,16 @@ fun ProtocolMediator.WifiScanResult.convert(): List<EspWifiItem>? {
     return accessPoints?.map {
         EspWifiItemImpl(it.ssid, it.bssid, it.rssi)
     }
+}
+
+fun DevInfo.toJson(): Map<String, String> {
+    return mapOf(
+        "devId" to devId,
+        "sku" to sku,
+        "snRadar" to snRadar,
+        "snProduct" to snProduct,
+        "hwRevRadar" to hwRevRadar,
+        "hwRevProduct" to hwRevProduct,
+        "swVer" to swVer
+    )
```

Comment:
"swVerCode": UInt32
"isProvisionComitted": bool

Reply:
These values were populated by the file Message (using the protobuf) I will cahnge the names also in the message file which we got from Vayyar